content/docs/started/workstation/getting-started-linux.md

```diff
@@ -0,0 +1,80 @@
++++
+title = "Kubeflow on Linux"
+description = "Install Kubeflow on Linux"
+weight = 2
++++
+
+<!--
+  TODO: Create a table that summarizes the options below, helping the user choose
+  more quickly
+-->
+
+## Introduction
```

Comment:
I think you can remove the heading "Introduction" and just have the introductory sentence as a nice lead-in to this page.

Reply:
Thx, removed.